atlas/map.go

```diff
@@ -59,6 +61,9 @@ type Map struct {
 
 	mvtProviderName string
 	mvtProvider     provider.MVTTiler
+
+	obsLck   sync.Mutex
```

Comment:
Should this lock be used? I don't see it being used anywhere in the diffs

Reply:
Good catch